app/controllers/profile.js

```diff
@@ -8,4 +10,285 @@ export default class ProfileController extends Controller {
   get imageUploadUrl() {
     return `${BASE_URL}/users/picture`;
   }
+
+  @tracked isSubmitDisabled = true;
+  @tracked isSubmitClicked = false;
+
+  @tracked title = 'Account Details';
+  @tracked formData = {
+    first_name: '',
+    last_name: '',
+    username: '',
+    email: '',
+    phone: '',
+    yoe: '',
```

Comment:
Since we are expecting `yoe` to be of type number.
Could we initialize this value with number please.
Ref: 
`
      id: 'yoe',
      label: 'Years of Experience',
      type: 'number',
      placeholder: 'How many years have you worked?',
      errorMessage: 'Number of years of experience is required',
`